iceoryx_posh/include/iceoryx_posh/internal/roudi/introspection/port_introspection.inl

```diff
@@ -481,6 +488,7 @@ void PortIntrospection<SenderPort, ReceiverPort>::PortData::prepareTopic(PortInt
 template <typename SenderPort, typename ReceiverPort>
 void PortIntrospection<SenderPort, ReceiverPort>::PortData::prepareTopic(PortThroughputIntrospectionTopic& topic)
 {
+    /// @todo #252 re-add port throughput for v1.0?
```

Comment:
I agree with you. Throughput calculation can easily be re-added in v1.x not a showstopper for v1.0

Reply:
Done.